mne/evoked.py

```diff
@@ -182,6 +181,12 @@ def __init__(self, fname, condition=None, baseline=None, proj=True,
                 elif my_kind == FIFF.FIFF_CH_INFO:
                     tag = read_tag(fid, pos)
                     chs.append(tag.data)
+                elif my_kind == FIFF.FIFF_FIRST_TIME:
+                    tag = read_tag(fid, pos)
+                    first_time = float(tag.data)
+                elif my_kind == FIFF.FIFF_NO_SAMPLES:
+                    tag = read_tag(fid, pos)
+                    nsamp = int(tag.data)
 
```

Comment:
@Eric89GXL out of curiosity is this where the reading magic happens? once the fiff constants and meas_info vars are defined.